classes/04class/exercises/c04-iam01/README.md

```diff
@@ -0,0 +1,30 @@
+# CLI + IAM: Create a new user with read only access to S3 (c04-iam01)
+
+Using ***only CLI commands***, do the following:
+
+1. Create a new IAM User via CLI called `user_readonly_S3`
+   1. Tip: `aws iam create-user help`
+2. Create a new Policy (via CLI) called `S3readOnly` that ALLOWS only S3 reads on every S3 resource. Take note of the created Policy ARN.
```

Comment:
you are already stating the the person need to use cli commands only in line 3. might not be needed to specify again.